app/models/certificate.rb

```diff
@@ -82,6 +86,10 @@ def for_surveys(surveys)
     end
   end
 
+  def init
+    self.attained_level ||= 'none'
+  end
+
```

Comment:
It's possible that might work in this case. The reason it is often safer to do it this way is because if you simply call `Certificate.new` without saving to the database `attained_level` won't yet have a value.